src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,11 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null || user.getPassword() == null || user.getAge() == null
+                || user.getAge() < 18 || user.getPassword().length() < 6
+                || storageDao.get(user.getLogin()) != null) {
+            throw new RuntimeException();
+        }
```

Comment:
let's split this if on several + when you throw an exception add informative message